src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/BuiltinUser.java

```diff
@@ -43,7 +44,7 @@
 
     @NotBlank(message = "Please enter a username.")
     @Size(min=2, max=60, message ="Username must be between 2 and 60 characters.")
-    @Pattern(regexp = "[a-zA-Z0-9\\_\\-\\.]*", message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
+    @ValidateUserName(message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
```

Comment:
Would be good to bundle-ize these messages if possible. Message added at line 186 in `Bundle.properties` is similar, could be neighbors.